src/components/map-container/map-container-component.scss

```diff
@@ -0,0 +1,22 @@
+#map {
+  display: block;
+  height: 300px;
+  width: 100%;
+  background-color: darkslategrey;
+}
+
+.marker {
+  // TODO: Replace this custom icon
+  background-image: url('../../images/mapbox-icon.png');
+
+  background-size: cover;
+  width: 50px;
+  height: 50px;
+  border-radius: 50%;
+  z-index: 40;
+  cursor: pointer;
+}
+
+.map-hidden-alert {
+  font-size: 12pt;
```

Comment:
I don't think I've seen `pt` before.

Reply:
Hah nice catch, typo.  `pt` is better for print, as 1pt = 1/72 of an inch.  When you're working in Word / GDocs, font size is in points.  

[Screens should always use pixels, percent, or ems,](https://kyleschaeffer.com/css-font-size-em-vs-px-vs-pt-vs-percent) because these use the actual screen or other css references as their reference point.